src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageSolutionDetailControlModel.cs

```diff
@@ -76,9 +96,14 @@ private void UpdateInstalledVersions()
                         project.InstalledVersion = installedVersion.PackageIdentity.Version;
                         hash.Add(installedVersion.PackageIdentity.Version);
                         project.AutoReferenced = (installedVersion as BuildIntegratedPackageReference)?.Dependency?.AutoReferenced == true;
+                        if (project.NuGetProject.ProjectStyle.Equals(NuGet.ProjectModel.ProjectStyle.PackageReference))
```

Comment:
use `using NuGet.ProjectModel`.
Don't fully qualify the enum

Reply:
I added a PR to improve our guidelines with this: https://github.com/NuGet/NuGet.Client/pull/3592

Reply:
You are already doing a project style check above right?

Reply:
I think you're asking what I asked here https://github.com/NuGet/NuGet.Client/pull/3555#discussion_r473109979
and now here https://github.com/NuGet/NuGet.Client/pull/3555#discussion_r473268210

Reply:
Yes, but this is for the package, here we are comparing if the package installed is from a PR project, otherwise the package doesn't have a requested version, because PC project don't use Allowed Version